query/common/time_series_result.go

```diff
@@ -73,3 +90,36 @@ func (r AQLTimeSeriesResult) SetHLL(dimValues []*string, hll HLL) {
 		}
 	}
 }
+
+// =====  Time series result methods end =====
+
+// =====  Non aggregate query result methods start =====
+
+// Append appends single value to specific row, result rows must be built in order
+func (r AQLQueryResult) Append(row int, value interface{}) error {
```

Comment:
likely we only need the interface to add row by row

Reply:
changed